packages/agoric-cli/src/deploy.js

```diff
@@ -401,89 +404,94 @@ export { bootPlugin } from ${JSON.stringify(absPath)};
           );
 
           const modulePath = pathResolve(moduleFile);
-          const mainNS = nativeEsm
-            ? await import(modulePath)
-            : esmRequire(modulePath);
+          let mainNS = await (nativeEsm && import(modulePath));
+          if (!mainNS) {
+            mainNS = esmRequire(modulePath);
+          }
           const main = mainNS.default;
           if (typeof main !== 'function') {
             console.error(
               `${moduleFile} does not have an export default function main`,
             );
-          } else {
-            await main(bootP, {
-              bundleSource: (file, options = undefined) =>
-                bundleSource(pathResolve(file), options),
-              cache,
```

Comment:
How does dropping `cache` relate to the other changes of this PR?

Reply:
That seems to be a remnant of a drive-by fix I did in my work tree. `cache` wasn't well-thought-out, and its (invalid) initialisation caused some unhandled rejections that worried our developers.  So it's gone for now.